File: java03t/src/main/java/jdbc/Test05.java
package jdbc;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.Statement;

public class Test05 {

  public static void main(String[] args) {
    Connection con = null;
    Statement stmt = null;
    
    try {
      DriverManager.registerDriver(new com.mysql.jdbc.Driver());
      
      con = DriverManager.getConnection(
        "jdbc:mysql://localhost:3306/java76db","java76", "1111");
      
      stmt = con.createStatement();
      
      // update 실행
      stmt.executeUpdate(
          "update board set title='okok', content='okokxxx' where bno=9");
      
      System.out.println("변경 성공!");
      
    } catch (Exception e) {
      e.printStackTrace();
      
    } finally {
      try {stmt.close();} catch (Exception e) {}
      try {con.close();} catch (Exception e) {}
    }
  }

}

File: node01t/src/step02/test05.js
// delete 실행 
var mysql = require('mysql'); 

var connection = mysql.createConnection({
  host     : 'localhost',
  user     : 'java76',
  password : '1111',
  database : 'java76db'
});

connection.connect();

// select 질의하기
connection.query(
  "delete from board where title=?", 
  ['okok'], 
  function(err, rows, fields) {
	  if (err) throw err;
	  
	  console.log("삭제 성공!");
});


connection.end();
